code/package.json

```diff
@@ -3,16 +3,25 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/pickers": "^3.3.10",
+    "@reduxjs/toolkit": "^1.5.1",
     "babel-eslint": "^10.0.3",
+    "date-fns": "^2.21.1",
     "eslint": "^6.7.1",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-react-hooks": "^2.3.0",
+    "moment": "^2.29.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.2.0"
+    "react-redux": "^7.2.3",
+    "react-scripts": "3.2.0",
+    "uniqid": "^5.3.0"
```

Comment:
If you want to clean your packege.json from unused dependencies: https://www.npmjs.com/package/depcheck